wallet/src/de/schildbach/wallet/BaseWalletApplication.kt

```diff
@@ -89,35 +73,9 @@ abstract class BaseWalletApplication : MultiDexApplication(), WalletDataProvider
         activity.startActivityForResult(sendCoinsIntent, requestCode)
     }
 
-    override fun sendCoins(address: Address, amount: Coin): LiveData<Resource<Transaction>> {
-        checkWalletCreated()
-        val wallet = walletApplication.wallet!!
-        val sendRequest = createSendRequest(address, amount)
-        val scryptIterationsTarget = walletApplication.scryptIterationsTarget()
-
-        return SendCoinsTask.sendCoins(wallet, sendRequest, scryptIterationsTarget)
-    }
-
-    override fun createSentDashAddress(address: String): Address {
-        return Address.fromString(Constants.NETWORK_PARAMETERS, address.toString().trim { it <= ' ' })
```

Comment:
This seems like a utility method that doesn't really belong here on in the `WalletDataProvider`. Implementation moved to the place of calling.